src/dbnode/storage/block/wired_list.go

```diff
@@ -75,7 +75,7 @@ var (
 
 // WiredList is a database block wired list.
 type WiredList struct {
-	sync.Mutex
+	sync.RWMutex
```

Comment:
make it a private variable - `mu sync.RWMutex`, otherwise the Lock/Unlock methods are exposed as part of public interface.